spec/components/candidate_interface/grouped_provider_courses_component_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe CandidateInterface::GroupedProviderCoursesComponent do
+RSpec.describe CandidateInterface::GroupedProviderCoursesComponent, mid_cycle: true do
```

Comment:
The second spec (of two) in this file explicitly travels to the correct point in the cycle — probably clearer to travel to `CycleTimetable.find_opens` in the first one?